### PR TITLE
Add image from URL

### PR DESCRIPTION
**Pull request by AraanBranco**
Add feature for load image from URL, issue #34

### CODE REVIEW

**Comment by khutten** [comment]
@AraanBranco Thanks a bunch for the PR. I'm a little busy atm, but I should be able to review this soon.

File: scripts.js [hunk 1]
```diff
@@ -28,10 +28,37 @@ document.getElementById('file-input').onchange = function (e) {
...
+  e.preventDefault();
+  const fileUrl = document.getElementById('file-url').value;
+  document.getElementById('file-url-error').style.display = "none";
+
+  // Check if url is valid
+  if (!domainValid(fileUrl)) {
+    document.getElementById('file-url-error').style.display = "block";
+  } else {
+    fetch(fileUrl)
+    .then(response => response.blob())
```
Comment (mrhut10) [inline scripts.js]: looks good, just tested going to a reasource that doesn't exist and it wont produce an error message in that case.
are you able to add a .catch to handle timeouts or/and handle not found response and add the error to that?

regards Alistair

**Comment by mrhut10** [comment]
thank you @AraanBranco

just had a very quick look just had a concern that we aren't handling not found errors or potentially timeouts if the server doesn't exist. are you able to handle these cases and add the error to your error display method on the field.

Regards from me :)

**Comment by AraanBranco** [comment]
I add one console for debbug error and alert user for error, I believe this will solve.

**Pull request merged**